unite@hardpixel.eu/panel.js

```diff
@@ -186,10 +186,13 @@ class WindowButtons extends Handlers.Feature {
     this.signals  = new Handlers.Signals()
     this.settings = new Handlers.Settings()
     this.styles   = new Handlers.Styles()
-    this.controls = new Buttons.WindowControls()
     this.themes   = new Theme.WindowControlsThemes()
     this.theme    = this.themes.default
     this.isDark   = true
+    this.controls = new Buttons.WindowControls({
+      iconScaleWorkaround: this.iconScaleWorkaround,
+      actionIcons: this.theme.getActionIcons(this.isDark)
+    })
```

Comment:
Prefer using `setControlThemeParams` to set these since it is a workaround and might be removed in the future.